example/contexts/FishjamExampleContext.tsx

```diff
@@ -78,7 +82,34 @@ const FishjamExampleContextProvider = (props: any) => {
   const [videoRoomState, setVideoRoomState] =
     useState<VideoRoomState>('BeforeMeeting');
 
+  const startForegroundService = async () => {
+    if (Platform.OS != 'android') return;
```

Comment:
I know that we only support two platforms, but I think it would be better to directly add condition for android here:
```suggestion
    if (Platform.OS === 'android') {
```